CHANGELOG.md

```diff
@@ -6,6 +6,174 @@
 
 Releases are now named after famous [Cricket](https://en.wikipedia.org/wiki/Cricket) players.
 
+## [0.5] - [Hamid Hassan](http://www.espncricinfo.com/afghanistan/content/player/311427.html) - 2017-11-??
+
+JupyterHub 0.8, HTTPS & scalability.
+
+### Upgrading from 0.4
+
+#### Database upgrade
+
+This release contains a major JupyterHub version bump (from 0.7.2 to 0.8). If
+you are using the default database provider (SQLite), then the required db upgrades
+will be performed automatically when you do a `helm upgrade`.
+
+If you are using MySQL / PostgreSQL as your backing database, you must perform a
+manual database upgrade before the helm upgrade.
```

Comment:
Let's note that neither of these are default behavior, so users should only need to do this if they explicitly specified one of those DBs in their helm chart. We could also link to https://zero-to-jupyterhub.readthedocs.io/en/v0.4-doc/reference.html?highlight=database#hub-db for more information. WDYT?

Reply:
Good catch @choldgraf. I think we should give a bit more guidance here. Perhaps the best route would be to be explicit:

```
This release contains a major JupyterHub version bump (from 0.7.2 to 0.8).

**Default (SQLite)**: The database upgrade will be performed automatically when you run `helm upgrade`.

1. Run `helm upgrade`. This will upgrade the database automatically.

**MySQL / PostgreSQL**: You will execute the following steps, which includes a manual update of your database:

...
```

I wouldn't link to the suggested section since it may cause more confusion. For guidance on upgrades of mysql/postgres, linking to the JupyterHub docs in Step 1 [full backup of your database](http://jupyterhub.readthedocs.io/en/latest/reference/upgrading.html). *Personally, I think this link would cause more confusion since the rest of the process is different.* For now, I would recommend leaving off additional links and amending the docs if needed later.

Reply:
+1 to this edit, I think that's a good point about unnecessary extra info

Reply:
I've done this now.